### PR TITLE
Streamline the CentOS/RHEL 5 kicks and use the repackaged nova-agent

### PR DESCRIPTION
**Pull request by shaynehardesty**
Removed unneeded systemd stuff, cleaned up a few things, and switched over to using the repackaged nova-agent, which will resolve the RPM db errors in RHEL 5

### CODE REVIEW

File: CentOS_5.cfg [hunk 1]
```diff
@@ -84,15 +79,6 @@ rpm -Uhv --nosignature nova-agent*.rpm
...
 
-# setup systemd to boot to the right runlevel
-echo -n "Setting default runlevel to multiuser text mode"
-rm -f /etc/systemd/system/default.target
-ln -s /lib/systemd/system/multi-user.target /etc/systemd/system/default.target
-
-# If you want to remove rsyslog and just use journald, remove this!
-echo -n "Disabling persistent journal"
-rmdir /var/log/journal/ 
-
```
Comment (broble) [inline CentOS_5.cfg]: curious, why these fixes?

File: CentOS_5.cfg [hunk 2]
```diff
@@ -135,13 +121,6 @@ cat > /etc/sysconfig/ip6tables <<'EOF'
 COMMIT
 EOF
 
-echo -n "Getty fixes"
-# although we want console output going to the serial console, we don't
-# actually have the opportunity to login there. FIX.
-# we don't really need to auto-spawn _any_ gettys.
-sed -i '/^#NAutoVTs=.*/ a\
-NAutoVTs=0' /etc/systemd/logind.conf
```
Comment (broble) [inline CentOS_5.cfg]: same here as above?

File: Red_Hat_Enterprise_Linux_5.cfg [hunk 3]
```diff
@@ -206,14 +180,6 @@ sed -i 's%#baseurl.*%baseurl=http://mirror.rackspace.com/epel/5/x86_64/%g' /etc/
...
 sed -i '/baseurl/s/# *//' /etc/yum.repos.d/epel.repo
 
-cat >> /etc/yum.repos.d/rhel-source.repo <<'EOF'
-[rhel-source]
-name=Red Hat Enterprise Linux $releasever - $basearch - Source
-baseurl=http://intra.mirror.rackspace.com/kickstart/rhel-x86_64-server-5/
-enabled=1
-gpgcheck=0
-EOF
-
```
Comment (broble) [inline Red_Hat_Enterprise_Linux_5.cfg]: same as above?

File: CentOS_5.cfg (hunk 1 above)
Comment (shaynehardesty) [inline CentOS_5.cfg]: These changes are specific to systemd, which doesn't exist in RHEL5.  I guess someone just applied them to all of the kicks at some point.

File: CentOS_5.cfg (hunk 2 above)
Comment (shaynehardesty) [inline CentOS_5.cfg]: These changes are specific to systemd, which doesn't exist in RHEL5.  I guess someone just applied them to all of the kicks at some point.

File: Red_Hat_Enterprise_Linux_5.cfg [hunk 4]
```diff
@@ -233,8 +199,7 @@ rm /boot/initrd-$version.img
 mkinitrd -v --with=xenblk --with=xennet --preload=xenblk --preload=xennet --omit-scsi-modules --omit-raid-modules /boot/initrd-$version.img $version
 
 # make sure repos are empty for rhel
-for k in $(find /etc/yum.repos.d -type f\( ! -name "*epel*" \)); do rm -f $k; done
-echo > /etc/yum.repos.d/rhel-source.repo
```
Comment (shaynehardesty) [inline Red_Hat_Enterprise_Linux_5.cfg]: That RHEL source repo gets added above, then truncated here.  As far as I can tell it's never used.  I think the only play we use it is in the VCloud image to build the vmnet drivers.

**Pull request merged**